docs/docs/configuration/cameras.md

```diff
@@ -130,37 +129,48 @@ objects:
+24/7 recordings can be enabled and are stored at `/media/frigate/recordings`. The folder structure for the recordings is `YYYY-MM/DD/HH/<camera_name>/MM.SS.mp4`. These recordings are written directly from your camera stream without re-encoding and are available in Home Assistant's media browser. Each camera supports a configurable retention policy in the config.
+
+Clips are also created off of these recordings. Frigate chooses the largest matching retention value between the recording retention and the event retention when determining if a recording should be removed.
 
-These clips will not be playable in the web UI or in Home Assistant's media browser unless your camera sends video as h264.
+These recordings will not be playable in the web UI or in Home Assistant's media browser unless your camera sends video as h264.
 
 :::caution
 Previous versions of frigate included `-vsync drop` in input parameters. This is not compatible with FFmpeg's segment feature and must be removed from your input parameters if you have overrides set.
 :::
 
 ```yaml
-clips:
-  # Required: enables clips for the camera (default: shown below)
-  # This value can be set via MQTT and will be updated in startup based on retained value
+record:
+  # Optional: Enable recording (default: shown below)
   enabled: False
-  # Optional: Number of seconds before the event to include in the clips (default: shown below)
-  pre_capture: 5
-  # Optional: Number of seconds after the event to include in the clips (default: shown below)
-  post_capture: 5
-  # Optional: Objects to save clips for. (default: all tracked objects)
-  objects:
-    - person
-  # Optional: Restrict clips to objects that entered any of the listed zones (default: no required zones)
-  required_zones: []
-  # Optional: Camera override for retention settings (default: global values)
-  retain:
-    # Required: Default retention days (default: shown below)
-    default: 10
-    # Optional: Per object retention days
+  # Optional: Number of days to retain (default: shown below)
+  retain_days: 0
+  # Optional: Event recording settings
+  events:
+    # Optional: Enable event recording retention settings (default: shown below)
+    enabled: False
+    # Optional: Maximum length of time to retain video during long events. (default: shown below)
+    # NOTE: If an object is being tracked for longer than this amount of time, the cache
```

Comment:
This is still true because the recording cleanup runs every minute and it checks the event table to see if there is any overlap. Since the event doesn’t get written until it ends, the cleanup will delete files that could potentially be associated to an event that hasn’t been saved yet. We use this config setting to determine how old recordings need to be to be considered for deletion, which is the earliest of either now - retain_days or now - this setting.